src/constraints/renewable_energy_constraints.jl

```diff
@@ -69,11 +73,21 @@ function add_re_elec_calcs(m,p)
     # Note: if battery ends up being allowed to discharge to grid, need to make sure only RE that is being consumed onsite is counted so battery doesn't become a back door for RE to grid.
 	# Note: calculations currently do not ascribe any renewable energy attribute to grid-purchased electricity
 
+	m[:AnnualGridREEleckWh] = @expression(m, p.hours_per_time_step * (
+			sum(m[:dvGridPurchase][ts, tier] * p.s.electric_utility.renewable_energy_fraction_series[ts] 
+				for ts in p.time_steps, tier in 1:p.s.electric_tariff.n_energy_tiers) # renewable energy from grid 
```

Comment:
It wouldn't change this calc since grid DV's are forced to zero during time steps without grid. Never mind though, this is a lower urgency project for another day.